src/SearchPanel.jsx

```diff
@@ -1,14 +1,37 @@
 import './SearchPanel.less'
+import { connect } from 'react-redux'
+import { FormattedDate } from 'react-intl'
 import PillPanel from './PillPanel'
 import React from 'react';
 import SearchBar from './SearchBar'
+import { shortIsoFormat } from './utils'
 
-export const SearchPanel = () =>
-    <div className="search-panel">
-      <h2>Search complaint data</h2>
-      <SearchBar />
-      <PillPanel />
-    </div>
+export class SearchPanel extends React.Component {
+  render() {
+    // Adjust UTC to local timezone
+    // This code adjusts for daylight saving time
+    // but does not work for locations east of Greenwich
+    var utcDate = new Date( this.props.lastUpdated )
+    var localTimeThen = new Date(
+      utcDate.getFullYear(),
+      utcDate.getMonth(),
+      utcDate.getDate() + 1
+    )
 
+    var tryagain = shortIsoFormat( utcDate );
 
-export default SearchPanel
+    return (
+      <div className="search-panel">
+        <h2>Search complaint data <span className="date-subscript">(last updated: <FormattedDate value={ tryagain } />)</span></h2>
```

Comment:
You should probably have a code path for when the last updated date is not available (like when the search fails, for example)

This will also help reduce the testing profile since the 'null' path would be executed by other components since they typically won't know to set up this variable